app/src/components/market/common/token_balance/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Wrapper = styled.div`
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+  padding: 0 20px;
+  height: 40px;
+  border-radius: 8px;
+  border: 1px solid ${({ theme }) => theme.borders.borderDisabled};
```

Comment:
i found about 16 instances where we use same  o "1px solid ${({ theme }) => theme.borders.borderDisabled " makes sense to have have that done same as cards border in theme.js file

Reply:
@Mi-Lan , borderColor is different from card border.

Reply:
@liaojikunwork i meant same principle to have all border parameters in one line and to reuse that instead of repeating this 1px solid ${({ theme }) => theme.borders.borderDisabled so it should look like this "border:borderLineDisabled" and borderLineDisabled:1px solid borders.borderDisabled; and this borderLine can be inside theme...